apps/studio/components/interfaces/Auth/Policies/Policies.tsx

```diff
@@ -37,7 +39,12 @@ const Policies = ({
   const { ref } = useParams()
   const { project } = useProjectContext()
 
-  const [selectedTableToToggleRLS, setSelectedTableToToggleRLS] = useState<any>({})
+  const [selectedTableToToggleRLS, setSelectedTableToToggleRLS] = useState<{
+    id: number
+    schema: string
+    name: string
+    rls_enabled: boolean
+  } | null>(null)
```

Comment:
just wondering - is it okay to use `undefined` here? or is `null` a better practice
cause we're currently using `undefined` as the default value in most places of the dashboard atm